parts/configure-swarmmode-cluster.sh

```diff
@@ -150,9 +151,13 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    curl --max-time 60 -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - 

```

Comment:
The packages are actually already installed on Ubuntu 16.04, but I've now added the command to always ensure the pre-reqs are there for HTTPS, in case they get removed in future images.